_includes/configuration.md

```diff
@@ -52,7 +62,7 @@ database.
 The `postgresql.conf` file used with {{ site.data.product.title_short }} databases requires
 specific settings for correct operation. For example, it must correctly
 reclaim table space, control session timeouts, and format the PostgreSQL
-server log for improved system support. It is recommended that external databases use a
+server log for improved system support. Due to these requirements, it is recommended that external {{ site.data.product.title_short }} databases use a
```

Comment:
We want to avoid adding product names where possible, so let's remove the ` {{ site.data.product.title_short }}`